src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,42 @@
+import { Person } from '../types';
+import { PersonInfo } from './PersonInfo';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const getPersonByName = (name: string | null): Person | null => (name
+    ? people.find(person => person.name === name) || null
+    : null
+  );
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
We can cheat a little and put the column names in an array outside the component and `.map` on it in the same way as we do with the `people` array.